### DISCUSSION TITLE
STL-any源码阅读

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#292] [Announcements]
https://bbing.com.cn/202207/cpp-stl-any/

**Comment by penndu** [comment]
学过一段时间的c++，最后只会写hello world。

**Comment by caibingcheng** [comment]
> @penndu
> 学过一段时间的c++，最后只会写hello world。

这些还是得做几个项目做才好掌握, 单纯学语法我容易过段时间就忘记了(比如学了几次Rust, 只跟着教程走, 现在是啥都不知道的状态)

**Comment by penndu** [comment]
> @caibingcheng
> > @penndu
> > 学过一段时间的c++，最后只会写hello world。
> 
> 这些还是得做几个项目做才好掌握, 单纯学语法我容易过段时间就忘记了(比如学了几次Rust, 只跟着教程走, 现在是啥都不知道的状态)

学完之后，最多就是改改！

**Comment by mysticalwing** [comment]
看得头疼

**Comment by caibingcheng** [comment]
> @qhw0
> 看得头疼

any里面manager的想法挺有意思的，最近看tuple的变参模板递归继承也比较有意思，正在整理，过两天发一下
